toolkit/quickrebuild_autoconfig.mk

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Contains:
+#	- build optimization flag auto configuration
+
+######## QUICKREBUILD AUTO CONFIGURE ########
+
+# The QUICKREBUILD* flags are special flags that will try to build the toolchain and packages as quickly as possible. They will
+# automatically set REBUILD_TOOLS, USE_CCACHE, REBUILD_TOOLCHAIN, DELTA_BUILD, INCREMENTAL_TOOLCHAIN, and ALLOW_TOOLCHAIN_DOWNLOAD_FAIL to 'y'.
```

Comment:
mostly an FYI here. I think this change is fine, but the request I was receiveing from others  was to have some sort of configuration file to read some of these items.  So there were simple defaults.

I do think part of the issue has been that folks who are not sure how to use the toolchain pass too many arguments that already default to the right things.  So, maybe we don't need a configuration file anyway.

Still, I think we should look for an opportunity here to eliminate unneeded arguments.  How many of these are really required.

Reply:
Would also love to have a config file, its listed in the tooling improvements document.